ios/browser/url_sanitizer/url_sanitizer_factory_wrappers.mm

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/ios/browser/url_sanitizer/url_sanitizer_factory_wrappers.h"
+
+#include "brave/components/url_sanitizer/browser/url_sanitizer_service.h"
+#include "brave/ios/browser/api/url_sanitizer/url_sanitizer_service+private.h"
+#include "brave/ios/browser/keyed_service/keyed_service_factory_wrapper+private.h"
+#include "brave/ios/browser/url_sanitizer/url_sanitizer_service_factory.h"
+#include "ios/chrome/browser/shared/model/browser_state/chrome_browser_state.h"
+
+#if !defined(__has_feature) || !__has_feature(objc_arc)
+#error "This file requires ARC support."
+#endif
+
+@implementation URLSanitizerFactory
```

Comment:
the naming convention here is problematic. The file name should match the class name, I also see this in other places that should be fixed like skus_sdk_factory_wrappers cc @kylehickinson 
This class name should also not change the name from `URLSanitizerServiceFactory` to `URLSanitizerFactory`
You could define both the cpp and obj in the same file url_sanitizer_service_factory.mm

Reply:
I don't know what to do here. The file name "url_sanitizer_service_factory.h" is already taken in the same folder, this is a objc file which represent the url_sanitizer_service_factory class but I can't expose this class directly to iOS hence this wrapper. I really don't think we want "Wrapper" at the end of our objc classes in iOS.